sycl/doc/OptionalDeviceFeatures.md

```diff
@@ -35,6 +36,31 @@ There are several categories of requirements covered by this design:
   speculative compilation of a kernel for a device, when the application does
   not specifically submit the kernel to that device.
 
+### Clarification of a kernel's static call tree
```

Comment:
Nit: In general case this is a graph rather than a tree. A -> B, A -> C, B -> D, C -> D

Reply:
I updated the wording to say "static call graph" instead of "tree".